### PR TITLE
#81921 Restore Functionality of $only_with_upcoming Parameter

### PR DESCRIPTION
**Pull request by ggwicz**
_Ref:_ [C#81921](http://central.tri.be/issues/81921)

### Code Review Note

After it was revealed that this is already fixed in the Full REST API bucket, we're *not* going to merge these changes. We're going to use Luca's fixes.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Each line in an array declaration must end in a comma | [venue.php:555](https://github.com/moderntribe/the-events-calendar/blob/51edbfc6a46a15b486ab9d079ed8516f21ef0a32/src/functions/template-tags/venue.php#L555)
:x: | Whitespace found at end of line | [venue.php:558](https://github.com/moderntribe/the-events-calendar/blob/51edbfc6a46a15b486ab9d079ed8516f21ef0a32/src/functions/template-tags/venue.php#L558)

> via jenkins/codesniffer-phpcs

**Comment by lucatume** [comment]
The same thing was done in the `bucket/full-rest-api` branch... https://github.com/moderntribe/the-events-calendar/blob/9a5bba522b81ca95cec5f662e0f46fd72cbe6d6f/src/functions/template-tags/venue.php#L395

**Comment by lucatume** [comment]
for organizers too: https://github.com/moderntribe/the-events-calendar/blob/9a5bba522b81ca95cec5f662e0f46fd72cbe6d6f/src/functions/template-tags/organizer.php#L333

**Comment by lucatume** [comment]
with tests for both in the REST API code: 
https://github.com/moderntribe/the-events-calendar/blob/df445168b7fc204114362802b5e24f9c7c5b1c4f/tests/wpunit/Tribe/Events/REST/V1/Endpoints/Archive_VenueTest.php#L188
https://github.com/moderntribe/the-events-calendar/blob/df445168b7fc204114362802b5e24f9c7c5b1c4f/tests/wpunit/Tribe/Events/REST/V1/Endpoints/Archive_OrganizerTest.php#L188

File: src/functions/template-tags/venue.php [hunk 1]
```diff
@@ -545,14 +545,50 @@ function tribe_get_phone( $postId = null ) {
...
+			'posts_per_page'   => $posts_per_page,
+			'suppress_filters' => $suppress_filters,
+			'post_status'      => 'publish'
 		);
 
+		if ( $only_with_upcoming ) {		
+			global $wpdb;
+
+			// Grabs all venue IDs that are associated with upcoming events.
+			$sql = "
```
Comment (bordoni) [inline src/functions/template-tags/venue.php]: I would say l go with what @lucatume did on the `bucket/rest-api`

**Comment by ggwicz** [comment]
@lucatume's is clearly the better code to use. so let's do that. what ticket number were those changes associated with, @lucatume?

**Pull request closed without merging**